### DISCUSSION TITLE
ERROR - Failed to execute job 1720641 for task get_mysql_metadata ((sqlite3.OperationalError) no such column: dag_run.queued_at

### DISCUSSION THREAD

**Discussion by kgowrisangar** [apache/airflow#24252] [General] [answered]
I have recently upgraded from the Airflow version 1.10.11 to 2.3.0. When I try to run a dag that was working fine in the earlier version, it's throwing me this error.

I also checked from the psql table whether this column is available, and it's there. Please check the attached SS.

![image](https://user-images.githubusercontent.com/59826267/172181570-e524362a-6c1a-4c13-bb88-a9b6ed2b973c.png)

Tried restarting the db but that did not work. Any help could be appreciated.

Python version - 3.7.3

Postgres server - 10.21

**Comment by potiuk** [comment, marked as answer]
I tihnk this might be the same reason as https://github.com/apache/airflow/issues/23679 - either move the deprecated configuration to "database" or migrate to Airlfow 2.3.2 (and then change the configuration to "database" as well.)

**Reply by potiuk** [reply]
But this is a wild guess

**Reply by kgowrisangar** [reply]
thank you @potiuk , let me try the first option and post the end result.

**Comment by kgowrisangar** [comment]
Thanks @potiuk  the change you suggested worked fine.

Added the [database] tag above the sqlalchemy connection details.

**Reply by potiuk** [reply]
Thanks for confirming. BTW 2.3.2 Should have it fixed regardless of using the deprecated config, and it is released on Saturday. Highly recommend to migrate - it fixes quite a number of similar "teething" problems with 2.3.0 and 2.3.1

**Reply by kgowrisangar** [reply]
Sure @potiuk will do. Thanks for the instant support
